config.yml

```diff
@@ -76,18 +76,21 @@ chrome:
         - npm --version
         - yarn --version
 
-dind-aws:
+dind-provider-cli:
   "1":
     build_args:
       CI_HELPER_VERSION: *CI_HELPER_VERSION
       DOCKER_COMPOSE_VERSION: 1.25.0
       GLIBC_VERSION: 2.27-r0
       PIP_VERSION: 18.1
       PIPENV_VERSION: 2018.10.13
+      IBM_CLI_VERSION: 0.22.0
+      AZURE_CLI_VERSION: 2.1.0
     test_config:
       cmd:
-        - *ci_helper_test
         - aws --version
+        - az version
+        - ibmcloud version
```

Comment:
Can you had a test on `ibmcloud cr` to check that CR plugin is well installed

Reply:
Fixed